### ISSUE TITLE
blockdiagcontrib-octicons - Cannot open ressource

### ISSUE THREAD

**Issue by tk0miya** [blockdiag/blockdiag-contrib#2]
Hi, 

I have following code:

```
# cat b
blockdiag {
  plugin octicons;

  A [label = "", background = "octicon://mark-github"];
}
```

`blockdiagcontrib-octicons` is installed in my virtualenv. When I try to generate the image I get:

```
# blockdiag b -T PNG -o b.png
WARNING: plugin "octicons" is already loaded. ignored.
ERROR: cannot open resource
```
Any ideas? 

Cheers,

Peter

----------------------------------------
- Bitbucket: https://bitbucket.org/blockdiag/blockdiag-contrib/issue/2
- Originally reported by: Peter Petersky
- Originally created at: 2014-11-19T11:58:40.216

**Comment by tk0miya** [comment]
_From Takeshi KOMIYA on 2011-12-12 09:34:49+00:00_

fixed at <<changeset 87ba94c9710a>>

**Comment by tk0miya** [comment]
_From Takeshi KOMIYA on 2015-06-12 13:34:48+00:00_

LGTM.
Could you send me pull request, please?
